lti_consumer/plugin/compat.py

```diff
@@ -108,17 +117,9 @@ def user_has_access(*args, **kwargs):
 def get_course_by_id(course_key):
     """
     Import and run `get_course_by_id` from LMS
-
-    TODO: Once the LMS has fully switched over to this new path [1],
-    we can remove the legacy (LMS) import support here.
-
-    - [1] https://github.com/edx/edx-platform/pull/27289
     """
     # pylint: disable=import-error,import-outside-toplevel
-    try:
-        from openedx.core.lib.courses import get_course_by_id as lms_get_course_by_id
-    except ImportError:
-        from lms.djangoapps.courseware.courses import get_course_by_id as lms_get_course_by_id
+    from openedx.core.lib.courses import get_course_by_id as lms_get_course_by_id
     return lms_get_course_by_id(course_key)
```

Comment:
I don't think the intention is to upgrade existing releases to this version. It's for next release of Open edX.

Reply:
@xitij2000, then we should consider bumping major, to indicate that it's not backward compatible change.

Reply:
Hmm. In that case, I think I'll prefer just removing this change. It was just a drive-by cleanup. I saw this code, saw the PR was merged so decided to get rid of the TODO.

Reply:
No need to do anything for Lilac then?

Reply:
@xitij2000, thanks. Sure, I know this was a cleanup, though it indeed sounds like a better idea to leave this as-is for now instead of reducing the compatibility.

@nedbat, as Kshitij has removed this change from the PR, no cherry-picks are needed for Lilac here.